### ISSUE TITLE
NVDA reading unknown in mozilla firefox and internet explorer

### ISSUE THREAD

**Issue by valentinvelecico** [nvaccess/nvda#6407]
I tried to teach a friend to use NVDA but we run into a very serious problem. NVDA doesn't read anything in internet explorer 10 or mozilla firefox the latest version. when I try to use the arrow keys or the tab key nothing is being read and when I switch windows NVDA keeps saying unknown instead of the name of the window. I have the latest version of NVDA and I tried everything I know including updating mozilla, doing a clean reinstal of nvda, multiple restarts and soon. I installed the screen reader on an other computer in the same house and I have the same problem. the computers are running windows 7 on 64 bit. Any ideas on how to solve this?

**Comment by jcsteh** [comment]
This is a duplicate of #2807. This is generally caused by a system component which has become unregistered. See #2807 for ideas regarding how this might be fixed.
